app/views/SidebarView/index.js

```diff
@@ -142,6 +154,17 @@ class Sidebar extends Component {
 		});
 	}
 
+	setStatusText = async(statusText) => {
+		const result = await RocketChat.saveUserProfile({ statusText }, {});
+		if (result.success) {
+			EventEmitter.emit(LISTENER, { message: I18n.t('Status_saved_successfully') });
+		} else {
+			EventEmitter.emit(LISTENER, { message: I18n.t('error-could-not-change-status') });
```

Comment:
See the above comment about consistency in the i18n string. We should make these consistent.

Reply:
Actually, scrap that. It looks like it is consistent with what is in the i18n file. Nevermind.